src/features/coursesFeatures/About/About.tsx

```diff
@@ -1,27 +1,35 @@
-import { Paragraph, Subtitle, Title } from '../../../../components';
```

Comment:
Each feature should reflect the root level structure: each should have its own "components" folder where all the components will live
check https://blog.stackademic.com/react-folder-file-structure-patterns-and-tips-part-1-b8e55bda446f (option 1, feature approach, option 3 there is similar to FSA, you can check it, but don't apply it to this project)

Reply:
ok